css/properties.json

```
@@ -2241,7 +2241,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/background-blend-mode"
   },
   "background-clip": {
-    "syntax": "<box>#",
+    "syntax": "[ <box> | text ]#",
```

```suggestion
    "syntax": "<bg-clip>#",
```

then add a [`bg-clip`](https://drafts.csswg.org/css-backgrounds-4/#typedef-bg-clip) syntax entry in *syntaxes.json*

the [`visual-box`](https://drafts.csswg.org/css-box-4/#typedef-visual-box) syntax entry is already added, so there is no use to adjust it